config/v1/tests/authentications.config.openshift.io/ExternalOIDC.yaml

```diff
@@ -120,6 +120,39 @@ tests:
               username:
                 claim: "preferred_username"
                 prefixPolicy: NoPrefix
+    - name: Can set OIDC providers with uid and extra mappings
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Authentication
+        spec:
+          type: OIDC
+          oidcProviders:
+          - name: myoidc
+            issuer:
+              issuerURL: https://meh.tld
+              audiences: ['openshift-aud']
+            claimMappings:
+              uid:
+                claim: claim
```

Comment:
According to the definition, `claim` is mutually exclusive with `expression` so we should add a separate test case for that, too.